debian/control

```diff
@@ -0,0 +1,20 @@
+Source: qubes-artwork
+Section: graphics
+Priority: extra
+Maintainer: Frédéric Pierret (fepitre) <frederic.pierret@qubes-os.org>
+Build-Depends:
+ debhelper (>= 10~),
+ inkscape,
+ imagemagick,
+ librsvg2-bin,
+ netpbm,
+ fonts-roboto,
+ qubes-utils,
+ python3-cairo
```

Comment:
`python3-cairo` should be rather in qubes-utils. See b71235cc

Reply:
https://github.com/QubesOS/qubes-linux-utils/pull/58